x-pack/plugins/security_solution/public/management/components/console/components/command_input/integration_tests/command_input.test.tsx

```diff
@@ -214,6 +224,44 @@ describe('When entering data into the Console input', () => {
         expect(getLeftOfCursorText()).toEqual('cmd1 --help');
       });
     });
+
+    it('should show confirm dialog when Clear history button is clicked', async () => {
+      await renderWithInputHistory('one');
+
+      userEvent.click(renderResult.getByTestId('test-clearInputHistoryButton'));
+
+      await waitFor(() => {
+        expect(renderResult.getByTestId('confirmModalTitleText'));
+      });
+    });
+
+    describe('and clear history confirm dialog is displayed', () => {
+      beforeEach(async () => {
+        await renderWithInputHistory('one');
+        userEvent.click(renderResult.getByTestId('test-clearInputHistoryButton'));
+        await waitFor(() => {
+          expect(renderResult.getByTestId('confirmModalTitleText'));
+        });
+      });
+
+      it('should close the confirm modal if Cancel button is clicked', async () => {
+        userEvent.click(renderResult.getByTestId('confirmModalCancelButton'));
+
+        await waitFor(() => {
+          expect(renderResult.queryByTestId('confirmModalTitleText')).toBeNull();
+        });
+      });
+
```

Comment:
I'll add an additional assertion to this test instead of a new one:

```
expect(renderResult.getByTestId('test-inputHistorySelector')).not.toBeNull();
```